include/kodlab_mjbots_sdk/joint_moteus.h

```diff
@@ -43,6 +31,30 @@ struct MoteusJointConfig{
 class JointMoteus: public JointBase
 {
     public:
+        /**
+         * @brief Define a default resolution struct
+         * 
+         */
+        static const ::mjbots::moteus::QueryCommand kDefaultQuery;
```

Comment:
@jdcaporale Do you think this is the correct place for these to live? Sort of torn if they should live here or in a namespace.

Reply:
Relocated to the bottom of the file for the time being, in keeping with the comparable change to source file.

Reply:
I think this is fine.